test-app/.env

```diff
@@ -0,0 +1,3 @@
+# Defines environment variables for docker-compose.
+# Can be overridden via e.g. `MARKLOGIC_TAG=latest-10.0 docker-compose up -d --build`.
+MARKLOGIC_IMAGE=progressofficial/marklogic-db:latest
```

Comment:
Based on the problems we had with this tag in the Jenkinsfile for the Spark connector - pinging @SameeraPriyathamTadikonda  to confirm - I think "11.3.0-ubi" should be the default instead of "latest".